src/app/app-home/app-home.component.html

```diff
@@ -32,7 +32,7 @@ <h1>
     </mat-toolbar>
 
     <mat-nav-list>
-      <mat-list-item routerLink="/app/home">
```

Comment:
please leave the `routerLink` just point it to `"/"` `<mat-list-item routerLink="/">`